static/js/shared/place_selector.tsx

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import _ from "lodash";
+import React, { useEffect, useState } from "react";
+import { Card, Container, CustomInput } from "reactstrap";
+
+import { SearchBar } from "../tools/timeline/search";
```

Comment:
should we move this to shared next? we should also rename it to "search_bar" to match the component.

Reply:
Agree with moving to shared, I'll do that in a separate PR. But also maybe "place_search" would make more sense? because it's just for searching places

Reply:
sure, that sounds good